hpx/traits/is_executor_parameters.hpp

```diff
@@ -23,37 +23,83 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
 {
     ///////////////////////////////////////////////////////////////////////////
     struct executor_parameters_tag {};
+    /// Parameters implement functions:
+    /// bool variable_chunk_size(Executor &)
+    /// std::size_t get_chunk_size(Executor &, F &&, std::size_t num_tasks)
+    struct executor_parameters_chunk_size_tag : executor_parameters_tag {};
+
+    /// Parameters implement functions:
+    /// void mark_begin_execution();
+    /// void mark_end_execution();
+    struct executor_parameters_mark_begin_end_tag : executor_parameters_tag {};
+
+    /// Parameters implement function std::size_t processing_units_count();
+    struct executor_parameters_processing_units_count_tag : executor_parameters_tag {};
+
+    /// Parameters implement function: void reset_thread_distribution(Executor &)
+    struct executor_parameters_reset_thread_distr_tag : executor_parameters_tag {};
 
     namespace detail
     {
         /// \cond NOINTERNAL
-        template <typename T>
+        template <typename T, typename Tag>
```

Comment:
Maybe the name is confusing, I changed it to have a general scheme for all types. I'm not sure, should I change the name to emphasize how we use detailed implementation?

Yes, we can change it, but it's not necessary - is_executor_parameters inherits from detailed version using both template types.